app/models/optOut/OptOutMessageResponse.scala

```diff
@@ -18,8 +18,28 @@ package models.optOut
 
 import models.incomeSourceDetails.TaxYear
 import models.itsaStatus.StatusDetail
+import play.api.libs.json.{Format, Json}
 
 case class YearStatusDetail(taxYear: TaxYear, statusDetail: StatusDetail)
 case class OptOutMessageResponse(oneYearOptOut: Boolean = false, taxYears: Array[TaxYear] = Array()) {
   def oneYearOptOutTaxYear: TaxYear = taxYears(0)
 }
+
+
+case class OptOutUpdateReason(code: Int) {
```

Comment:
good candidate to use value class instead of plain case class

Reply:
turns out I don't need it, but will do if I introduce it again